source/NVDAObjects/window/winword.py

```diff
@@ -558,6 +558,31 @@ def collectionFromRange(self,rangeObj):
 	def filter(self,item):
 		return item.type==wdInlineShapeChart
 
+
+class LazyControlField(textInfos.ControlField):
+
+	def __init__(self, ti):
+		self._ti = ti

```

Comment:
Have you considered using a weak reference here?

Reply:
It is possible that the TextInfo may die while the ControlField is still alive, as the TextInfo may only be a local variable where it is used in both speech and braille.